scripts/infra_tools/README.md

```diff
@@ -0,0 +1,6 @@
+### Contents of dir
```

Comment:
Should start with one #

Reply:
i made it heading 3 and not heading 1 for now